pkg/virt-handler/vm.go

```diff
@@ -1420,6 +1420,7 @@ func getDomain(vmi *v1.VirtualMachineInstance) (*api.Domain, error) {
 		IsBlockDV:      isBlockDVMap,
 		// we won't use the domain object for actual libvirt definition
 		UseEmulation: true,
+		BestEffort:   true,
```

Comment:
And this one? I think it would deserve a docstring with explanation what will be gracefuly ignored in this mode.